### DISCUSSION TITLE
Using form attribute

### DISCUSSION THREAD

**Discussion by scottiemc7** [bigskysoftware/htmx#1326] [Q&A] [answered]
Hi,

I have a pretty simple page setup with some text filters in a form on top with search results below and some pagination controls below that.

![Untitled-2023-03-09-0915-2](https://user-images.githubusercontent.com/1521455/224850551-f661b775-29fd-4460-9a0b-84b50b1cb73a.png)

I'd like to include the pagination controls in the search form. In order to place them at the bottom of the page I'm using the form attribute.
`<button type="submit" form="searchForm">Next</button>`

This all works fine until I start to progressively enhance with htmx. Actually, this works great:
`<button type="submit" hx-get="/Search" hx-swap="none" hx-include="#searchForm">Next</button>`

But this causes htmx to abort the request:
`<button type="submit" hx-get="/Search" hx-swap="none" hx-include="#searchForm" form="searchForm">Next</button>`

I'm really unsure why. I get an error at `htmx:afterRequest` but there are no details. [Here's a fiddle](https://jsfiddle.net/0b84mL3s/). Is there a good way to use the form attribute without causing issues? I realize I could wrap the entire page in a form and that would solve the problem, but the records returned have forms in them so I'd like to avoid that.

Thanks

**Comment by Telroshan** [comment, marked as answer]
Hey @scottiemc7, very late reply, but could you tell me if you still encounter the issue with the latest htmx version?
The [1.9.4](https://github.com/bigskysoftware/htmx/releases/tag/v1.9.4) release had a [fix for buttons & form attribute](https://github.com/bigskysoftware/htmx/pull/1559), which might have solved your problem there

**Reply by scottiemc7** [reply]
Thanks @Telroshan. I will try this tonight and see if the issue is resolved.

**Reply by Telroshan** [reply]
Hey, did you have the chance to test this @scottiemc7 ?

**Reply by scottiemc7** [reply]
Hi @Telroshan,

I just looked at this and I don't think there is an issue. Thanks for your help!
